### ISSUE TITLE
Issue with TeamviewerExporter

### ISSUE THREAD

**Issue by MarkJAllDay247** [MyUncleSam/TeamviewerExporter#1]
When I run this program I get the following error message
![image](https://user-images.githubusercontent.com/43274584/46018383-5bf64780-c09f-11e8-9b8e-879817ddf8d5.png)
Is there a way to write the file to some other location?

**Comment by MyUncleSam** [comment]
@MarkJAllDay247 the program saves the export files in the directory where the .exe is in (could also be that it saves inside the working directory - not sure).

So please make sure that the program have write permission to that directory! At the moment it looks like it want to save the files into C:\Windows\original.csv which is prohibited by default. Additional I have no idea what it does if you execute it from a network path.

There is no possibility to change the location at the moment.

**Comment by MarkJAllDay247** [comment]
So I was able to figure it out. I run a Mac with Parallels, I saved the files to my desktop (See Screen Shot below). After updating my TV Token and running the TeamViewerExporter.exe it ran without any issues. I was running into issues because I was running it from my Mac side, like I said once I switched to my Windows 10 environment it worked just fine. Thanks for creating this, this helped us A LOT!!!
